sagemcom_api/client.py

```diff
@@ -288,6 +288,15 @@ async def login(self):
         else:
             raise UnauthorizedException(data)
 
+    async def logout(self):
+        actions = {
+            "id": 0,
```

Comment:
I need to check if we should remove the `id`, but I see we use this in a few other requests as well..

Reply:
Also, when I logout of the router, it sends the request with "id":0 included.

Reply:
I wonder if we should auto add / increment this id from the request generator, instead of here. But that is for later.